test/functional/eval/null_spec.lua

```diff
@@ -60,7 +60,7 @@ describe('NULL', function()
     -- FIXME should be accepted by writefile(), return {0, {}}
     null_expr_test('is accepted as an empty list by writefile()',
                         ('[writefile(L, "%s"), readfile("%s")]'):format(tmpfname, tmpfname),
-                        'E484: Can\'t open file ' .. tmpfname, {0, {}})
+                        'E484: Cannot open file ' .. tmpfname, {0, {}})
```

Comment:
It seems to be a localization thing.  A quick fgrep for `E484: ` shows both spellings.  Tests that check error messages probably should explicitly force a locale for messages, in the test environment.  Mine is:
```
JdeBP %locale
LANG=
LC_CTYPE="C"
LC_COLLATE="C"
LC_TIME="C"
LC_NUMERIC="C"
LC_MONETARY="C"
LC_MESSAGES="C"
LC_ALL=
JdeBP %
```